src/pages/checkout/payment-method.js

```diff
@@ -0,0 +1,234 @@
+// Libraries
+import React, { Component } from 'react'
+import Router from 'next/router'
+
+// Actions
+import { 
+  setPaymentMethod,
+  setCheckoutShippingAddress,
+  setCheckoutBillingAddress,
+  setPayPalOrderDetails
+} from '../../actions/checkout-actions'
+
+import { 
+  setCartBillingAddress,
+  createBillingAddress,
+  setCartShippingAddress,
+  createShippingAddress
+} from '../../actions/cart-actions'
+
+// Components
+import { PaymentMethods, Loading } from '@shiftcommerce/shift-react-components'
+
+export class PaymentMethodPage extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      loading: false
+    }
+    this.nextSection = this.nextSection.bind(this)
```

Comment:
`nextSection` doesn't use `this` so, technically, this binding is redundant 🤔

Reply:
We are passing it to the `PaymentMethods` component